ravendb/rvn_server.go

```diff
@@ -403,6 +478,9 @@ func (sc *ServerConfig) deployServer(publicIP string, index int) (err error) {
 	settings["ServerUrl.Tcp"] = "tcp://0.0.0.0:" + strconv.Itoa(sc.Url.TcpPort)
 	settings["Setup.Mode"] = "None"
 	settings["License.Path"] = "/etc/ravendb/license.json"
+	settings["Security.AuditLog.RetentionTimeInHours"] = "52560"
+
+	delete(settings, "Security.AuditLog.RetentionTimeInHrs")
```

Comment:
?? Why you set and remove it?

Reply:
`RetentionTimeInHrs` vs `RetentionTimeInHours`. 
`RetentionTimeInHrs` casues `404` in the studio as I mentioned in the PR description. 
Verifyed with @Danielle9897 and that's the reason for the change.

- Adding both of `Hrs` and `Hours` will solve it.
- Adding only `Hours` solve this as well.

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-17777
https://issues.hibernatingrhinos.com/issue/RavenDB-17779